File: Ejercicio_1.java
import java.util.Scanner;
class Main {
  public static void main(String[] args) {
    float t,p;
    String d,tu;
    Scanner lectura = new Scanner(System.in);
    
    System.out.println("Ingrese los minutos completos al telefono: ");
    t = lectura.nextFloat();
    System.out.println("Ingrese el dia de la llameda(LUN,MAR,MIER,JUE,VIE,SAB,DOM)");
    d = lectura.next();
    System.out.println("Ingrese el turno correspondiente: (matutino = M ; vespertino = v)");
    tu = lectura.next();

    if(t<=5){        
        p=1;
    }else if(t<8){
        p=(float)((t-5)*0.8)+5;
    }else if(t<10){
        p=(float)((t-8)*0.7 + 7.4);
    }else{
        p=(float)((t-10)*0.5);
    }

    if(d=="DOM"){
        p=(float)(p+p*0.05);
    }else if(tu=="M"){
        p=(float)(p+p*0.15);
    }else{
        p=(float)(p+p*0.10);
    }
    
    System.out.println("El valor de la llamada es: " + p);
  }
}

File: Ejercicio_1.cpp
#include "iostream"
#include "string"
using namespace std;

int main(){	

	float t, p;
	string d,tu;
	
	cout<<"Ingrese los minutos completos al telefono: \n"<<endl;
	cin>>t;
	cout<<"Ingrese el dia de la llameda(LUN,MAR,MIER,JUE,VIE,SAB,DOM)\n"<<endl;
	cin>>d;
	cout<<"Ingrese el turno correspondiente: (matutino = M ; vespertino = v)\n"<<endl;
	cin>>tu;
	
	if(t<=5){        
        p=1;
    }else if(t<8){
        p=(t-5)*0.8+5;
    }else if(t<10){
        p=(t-8)*0.7 + 7.4;
    }else{
        p=(t-10)*0.5;
    }

    if(d=="DOM"){
        p=p+p*0.05;
    }else if(tu=="M"){
        p=p+p*0.15;
    }else{
        p=p+p*0.10;
    }
    
    cout<<"\nEl valor de la llamda es: "<<p<<endl;
    
}
